src/include/api.h

```diff
@@ -87,10 +87,12 @@
 
 #define API_CALL_NOCONF(s, struct_name, func_name, dh) \
     do {                                               \
+        bool __set_err = true;                         \
     API_SESSION_INIT(s, struct_name, func_name, dh)
```

Comment:
nit: indentation was off in the original code
```suggestion
        API_SESSION_INIT(s, struct_name, func_name, dh)
```

Reply:
It is a bug in s_all. It formats it this way.